docs/REST API/rest-api.md

```diff
@@ -893,6 +893,68 @@ Requests a base64 encoded certificate (`.pem`). Currently, this endpoint is only
 
 ---
 
+## Conditional access
+
+- [Get Okta certificate](#get-okta-certificate)
+- [Get Okta configuration profile](#get-okta-configuration-profile)
+- [Delete Microsoft Entra ID](#disconnect-microsoft-entra-id) 
+
+### Get Okta certificate
+
+Download the SAML IdP signing certificate for Okta conditional access. Okta uses this certificate to verify SAML assertions signed by Fleet.
+
+`GET /api/v1/fleet/conditional_access/idp/signing_cert`
+
+#### Parameters
+
+None.
+
+#### Example
+
+`GET /api/v1/fleet/conditional_access/idp/signing_cert`
+
+##### Default response
+
+`Status: 200`
+
+Returns a PEM-encoded X.509 certificate file with `Content-Type: application/x-pem-file`.
+
+### Get Okta configuration profile
+
+Download the Apple configuration profile (.mobileconfig) for Okta conditional access. This profile configures SCEP enrollment and client certificate authentication on Apple devices.
+
+`GET /api/v1/fleet/conditional_access/idp/apple/profile`
+
+#### Parameters
+
+None.
+
+#### Example
+
+`GET /api/v1/fleet/conditional_access/idp/apple/profile`
+
+##### Default response
+
+`Status: 200`
+
+Returns an Apple configuration profile file with `Content-Type: application/x-apple-aspen-config`.
+
+### Delete Microsoft Entra ID
+
+Fleet will be disconnected from Microsoft Entra and will stop blocking end users from logging in with single sign-on.
```

Comment:
```suggestion
Disconnects Fleet from Microsoft Entra. This means end users, whose hosts are failing policies, will no longer be blocked from logging in to third-party apps.
```

@getvictor does deleting Microsoft Entra ID unblock end users right away? Or, does the end user have to wait for the next Fleet scheduled job to run?

Reply:
I'll check, yes. Approving this PR before I check b/c I don't think the check blocks use from merging.